ci/ios/buildserver-build-ios.sh

```diff
@@ -93,7 +93,7 @@ function run_build_loop() {
 
         run_git fetch --prune --tags 2> /dev/null || continue
         local tags
-        tags=( $(run_git tag | grep "$TAG_PATTERN_TO_BUILD") )
+        mapfile -t tags < <(run_git tag | grep "$TAG_PATTERN_TO_BUILD")
```

Comment:
https://www.shellcheck.net/wiki/SC2207

Reply:
note: only supported on bash 4.4+, but that was released in 2017 so should be fine.
also not posix compatible, but I assume that's not something we care about